src/main/resources/application-uat.yml

```diff
@@ -1,5 +1,17 @@
 gpfd:
   redirect-uri-template: https://laa-get-payments-finance-data-uat.apps.live.cloud-platform.service.justice.gov.uk/login/oauth2/code/gpfd-azure-uat
+  # DEV MOJFIN database config - needs to change to UAT when that exists:
+  datasource:
```

Comment:
We will also need this adding to application-prod.yml

Reply:
We don't currently have one as we haven't done a prod deployment :)